flickit-assessment-core/src/test/java/org/flickit/assessment/core/application/service/assessmentuserinvitee/InviteAssessmentUserServiceTest.java

```diff
@@ -0,0 +1,192 @@
+import org.flickit.assessment.common.exception.ResourceNotFoundException;
+import org.flickit.assessment.core.application.domain.AssessmentUserRole;
+import org.flickit.assessment.core.application.domain.User;
+import org.flickit.assessment.core.application.port.in.assessmentinvitee.InviteAssessmentUserUseCase.*;
+import org.flickit.assessment.core.application.port.mail.SendFlickitInviteMailPort;
+import org.flickit.assessment.core.application.port.out.assessment.CheckAssessmentSpaceMembershipPort;
+import org.flickit.assessment.core.application.port.out.assessment.GetAssessmentPort;
+import org.flickit.assessment.core.application.port.out.assessmentinvitee.InviteAssessmentUserPort;
+import org.flickit.assessment.core.application.port.out.assessmentuserrole.GrantUserAssessmentRolePort;
+import org.flickit.assessment.core.application.port.out.assessmentuserrole.LoadUserRoleForAssessmentPort;
+import org.flickit.assessment.core.application.port.out.space.InviteSpaceMemberPort;
+import org.flickit.assessment.core.application.port.out.spaceuseraccess.CheckSpaceAccessPort;
+import org.flickit.assessment.core.application.port.out.spaceuseraccess.CreateAssessmentSpaceUserAccessPort;
+import org.flickit.assessment.core.application.port.out.user.LoadUserPort;
+import org.flickit.assessment.core.test.fixture.application.AssessmentMother;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.flickit.assessment.common.error.ErrorMessageKey.COMMON_CURRENT_USER_NOT_ALLOWED;
+import static org.flickit.assessment.core.common.ErrorMessageKey.ASSESSMENT_ID_NOT_FOUND;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class InviteAssessmentUserServiceTest {
+
+    @InjectMocks
+    private InviteAssessmentUserService service;
+
+    @Mock
+    private GetAssessmentPort getAssessmentPort;
+
+    @Mock
+    private LoadUserPort loadUserPort;
+
+    @Mock
+    private LoadUserRoleForAssessmentPort loadUserRoleForAssessmentPort;
+
+    @Mock
+    private InviteSpaceMemberPort inviteSpaceMemberPort;
+
+    @Mock
+    private InviteAssessmentUserPort inviteAssessmentUserPort;
+
+    @Mock
+    private SendFlickitInviteMailPort sendFlickitInviteMailPort;
+
+    @Mock
+    GrantUserAssessmentRolePort grantUserAssessmentRolePort;
+
+    @Mock
+    CheckAssessmentSpaceMembershipPort checkAssessmentSpaceMembershipPort;
+
+    @Mock
+    CheckSpaceAccessPort checkSpaceAccessPort;
+
+    @Mock
+    CreateAssessmentSpaceUserAccessPort createAssessmentSpaceUserAccessPort;
+
+    @Test
+    @DisplayName("If the assessment is not exists, the service should throw a not found exception.")
```

Comment:
Change to `If the assessment does not exist, the service should throw a notFoundException.`